specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json

```diff
@@ -1900,8 +1910,22 @@
       "description": "Parameters supplied to the Create or Update Event Source operation for an IoTHub event source."
     },
     "EventSourceUpdateParameters": {
+      "discriminator": "kind",
```

Comment:
Add this to discriminate "EventHubEventSourceUpdateParameters" and "IoTHubEventSourceUpdateParameters".